frontend/pages/[resourceType]/[id].tsx

```diff
@@ -0,0 +1,74 @@
+import { Prism } from '@mantine/prism';
+import { Divider, Group, ScrollArea, Stack, Tabs, Text } from '@mantine/core';
+import { GetServerSideProps } from 'next';
+import { useRouter } from 'next/router';
+import BackButton from '../../components/BackButton';
+
+/**
+ * Component which displays the JSON/ELM/CQL/narrative content of an individual resource using
+ * Mantine tabs
+ * @returns JSON/ELM/CQL/narrative content of the individual resource in a Prism component
+ */
+export default function ResourceIDPage({ jsonData }: { jsonData: fhir4.Measure | fhir4.Library }) {
+  const router = useRouter();
+  const { resourceType, id } = router.query;
```

Comment:
Now that we have the type safety of the resource, it's probably best to remove these entirely and just render `jsonData.resourceType` and `jsonData.id` instead. That way we don't have to wait for the client to catch up and parse the router query in order to render that information on the page.